### ISSUE TITLE
Google provider -> IndexError: list index out of range

### ISSUE THREAD

**Issue by ziotibia81** [DenisCarriere/geocoder#104]
Searching the address "PIAZZA FIDIA, 1 - 20159 MILANO (MI)" using google provider I have this error:

Traceback (most recent call last):
  File "geocode.py", line 19, in <module>
    g = geocoder.google(addr,  proxies=prox)
  File "/home/macro/geo/geocoder-master/geocoder/api.py", line 83, in google
    return get(location, provider='google', **kwargs)
  File "/home/macro/geo/geocoder-master/geocoder/api.py", line 69, in get
    return options[provider][method](location, **kwargs)
  File "/home/macro/geo/geocoder-master/geocoder/google.py", line 66, in __init__
    self._initialize(**kwargs)
  File "/home/macro/geo/geocoder-master/geocoder/base.py", line 88, in _initialize
    self._json()
  File "/home/macro/geo/geocoder-master/geocoder/base.py", line 94, in _json
    value = getattr(self, key)
  File "/home/macro/geo/geocoder-master/geocoder/google.py", line 104, in quality
    return self.parse['types'][0]
IndexError: list index out of range

**Comment by DenisCarriere** [comment]
Thanks for the report, I'll send an fix tonight.

**Comment by DenisCarriere** [comment]
I've added the fix in the new version 1.0.8, you should be able to upgrade your version using PyPi.

```bash
$ pip install geocoder --upgrade
```
The response of your query gave me the following:
```json
{
    "status": "OK", 
    "city": "Milano", 
    "confidence": 9, 
    "ok": true, 
    "country": "Italy", 
    "provider": "google", 
    "location": "PIAZZA FIDIA, 1 - 20159 MILANO (MI)", 
    "county": "Milano", 
    "state": "Lombardia", 
    "bbox": {
        "northeast": [
            45.49068248029149, 
            9.187229330291503
        ], 
        "southwest": [
            45.48798451970849, 
            9.184531369708498
        ]
    }, 
    "address": "Piazza Fidia, 20159 Milano, Italy", 
    "lat": 45.4893578, 
    "lng": 9.1858509, 
    "postal": "20159", 
    "accuracy": "APPROXIMATE"
}
```
